### ISSUE TITLE
Done button does not show in the Add pilot screen

### ISSUE THREAD

**Issue by huguesfcadets** [cadets-ca/ets#67]
Assyst ticket 8153199.

It does not show in any iPad version with any iOS version.

It does show on iPhone in portrait. Not in landscape.

**Comment by kirvanp** [comment]
This sounds like the traitCollection bug. I believe that my last submission fixes it.

**Comment by huguesfcadets** [comment]
I integrated your previous changes, and unfortunately it does not fix that issue... Need to do more testing. Maybe I made an error while integrating your changes. I remember having a lot of trouble with that show or hide function. I though I fixed it, but apparently I made it worst... It is not clear why it uses the traitCollection.horizontalSizeClass to decide wether or not to display the Done button (or dismiss or other similar buttons).
For reference here is when horizontalSizeClass is compact vs Regular, it appears to me that it is not what we should base the decision to display the button or not. Any thoughts?

![](https://docs-assets.developer.apple.com/published/ec54d1af61/5a654d46-f860-41c4-853f-2307581a3c33.png)
